src/JuMP.jl

```diff
@@ -836,6 +922,8 @@ end
 
 ##########################################################################
 # Operator overloads
+typealias JuMPTypes Union(Variable,Norm,AffExpr,QuadExpr,SOCExpr)
```

Comment:
No it's justified, but this is pretty much only used for array operations. We can probably make it ``Generic`` without any trouble.